src/components/ListItem.jsx

```diff
@@ -1,5 +1,72 @@
 import './ListItem.css';
+import { updateItem } from '../api';
+import { useState, useEffect } from 'react';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({ item, listPath }) {
+	const { id, name, dateLastPurchased } = item;
+
+	const currentDate = new Date();
+	const lastPurchasedDate = dateLastPurchased
+		? new Date(dateLastPurchased.seconds * 1000)
+		: 0;
+	const timeDiff = currentDate - lastPurchasedDate; //milliseconds
+	const oneDayInMilliseconds = 24 * 60 * 60 * 1000;
+	const [isChecked, setIsChecked] = useState(false);
+
+	useEffect(() => {
+		if (timeDiff < oneDayInMilliseconds) {
+			setIsChecked(true);
+		}
+
+		let timeoutId;
```

Comment:
Question: current empty declaration of timeoutId works. We did some tests and wonder if declaring it on line 24 as a const would make it so the data type cannot be changed. Doesn't seem pressing but something that did pop out to us.

Reply:
Thanks for bringing this up! 🙏🏻 This was more a reflection of the thought process as the setTimeout function was pieced together rather an intentional separation of declaration. I agree with the 'const' suggestion, will update.